wpk/macos/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM debian:9
```

Comment:
is it required? couldn't we reuse the windows one?

Reply:
Done.
I reused the Windows Dockerfile and renamed the folder to "common".